### ISSUE TITLE
Stuttering motion with multiple characters with 'stray' behavior

### ISSUE THREAD

**Issue by AndyTechGuy** [Terasology/Behaviors#15]
With multiple characters using the same `stray` behavior, a target update on one character updates the targets of all stray characters. This can be seen in the gif below, with insensitiveDeer characters from [Terasology/WildAnimals](https://github.com/Terasology/WildAnimals):

![deerMovement](https://user-images.githubusercontent.com/44126397/59575337-a0929a80-9095-11e9-9fc7-8bbc58a0a21d.gif)

The stray behavior works perfectly with only one character, but with more and more characters the movement becomes more and more erratic.

**Comment by dkambersky** [comment]
Update: I had a hunch and now am pretty sure that this is a duplicate / symptom of #6 . Still looking into it. Ugly but, looking through a lot of levels of abstraction / delegation which makes stuff pretty hard to debug.

One thing you can't say is that Synopia underengineered things :P

**Comment by dkambersky** [comment]
So while this one is fixed (I had it fixed for ages in my version, just for some reason it never made its way upstream), the underlying issue likely isn't. Let me know in #6 if you find any other interesting ways to reproduce it. ( @casals you said you ran into this at some point too?)

tl;dr the outer 'loop' node is useless and breaks things, don't use it. Still no reason it should break things this way though, so this isn't entirely fixed.
